team-validator.js

```diff
@@ -214,6 +215,14 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		if (tools.getNature(set.nature).id && !tools.getNature(set.nature).name.exists) {
```

Comment:
If you're storing everything in `set.nature` anyway, what's wrong with `if (set.nature && !set.nature.exists)`?